db/migrate/20220324220835_add_users_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :users, null: false, foreign_key: true
```

Comment:
- [ ] Following the requirement as shown in the ERD. It will be nice to replace `:users` with `:author` as the foreign_key
![image](https://user-images.githubusercontent.com/51296741/160021718-70c38145-0c46-4169-8401-d39698938c86.png)

Reply:
This issue is well resolved! ✔️